torchtune/models/llama3/_tokenizer.py

```diff
@@ -25,7 +25,7 @@
     "<|eom_id|>": 128008,
     "<|eot_id|>": 128009,
     "<|python_tag|>": 128010,
-    "<|image|>": 128011,
+    "<|image|>": 128256,
```

Comment:
have we verified that this the official image token id...? When I first added this it was 128011

Reply:
Long story, but both numbers were supposed to be correct. One for finetuning and one for inference, but now it's just the one.